src/cgroup.rs

```diff
@@ -588,6 +589,33 @@ pub fn get_cgroups_relative_paths_by_pid(pid: u32) -> Result<HashMap<String, Str
     get_cgroups_relative_paths_by_path(path)
 }
 
+fn get_cgroup_destination(mut mount_root: String, pidpath: String) -> String {
```

Comment:
the parameter mount_root and pidpath can be &str?

Reply:
Yeah, i can update it here.